src/dataLabel/dataLabelUtils.ts

```diff
@@ -23,6 +23,7 @@
 *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 *  THE SOFTWARE.
 */
+import * as d3 from "d3";
```

Comment:
Are sure that it's good idea to import everything? Will webpack include whole d3 in such case?